pallets/treasury-buyout-extension/src/mock.rs

```diff
@@ -152,6 +165,64 @@ impl orml_currencies::Config for Test {
 	type WeightInfo = ();
 }
 
+impl orml_asset_registry::Config for Test {
```

Comment:
That's right! Removed it now since initially I thought that we should have asset registry directly as a config bound for the pallet and I forgot to remove it from the mock.